projects/demo/src/modules/components/reorder/index.html

```diff
@@ -14,5 +14,33 @@
         />
     </ng-template>
 
-    <tui-setup *pageTab="'Setup'" />
+    <ng-template pageTab="Setup">
+        <ol class="tui-list tui-list_ordered">
+            <li class="tui-list__item">
+                <p>Add an import:</p>
+
+                <tui-doc-code [code]="importCode" />
+            </li>
+
+            <li class="tui-list__item">
+                <p>Add to the template:</p>
+
+                <tui-doc-code [code]="templateCode" />
+            </li>
+
+            <li class="tui-list__item">
+                <p>
+                    By default reordering has swap behaviour. You can overwrite it by providing either
```

Comment:
I don't think we ever need to override it in `Reorder`. It makes sense for it to always be in `shift` mode